net.sourceforge.jpdftweak.jPdfTweak.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id" : "net.sourceforge.jpdftweak.jPdfTweak",
+  "branch" : "stable",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "19.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk11" ],
+  "command": "net.sourceforge.jpdftweak.jPdfTweak",
+  "finish-args" : [
+    "--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+	"--socket=fallback-x11",
```

Comment:
you need to have either socket=x11 or fallback-x11, not both. Also, the indenting.

If this works perfectly under wayland then you likely want fallback-x11.

Reply:
I tested ... and I need `socket=x11`, so I removed "fallback-x11" and a few other permissions (like `network`).

Reply:
Do you mean it doesn't work at all under Wayland? If so you need to remove that permission, wayland users should fall back to XWayland fine.

Reply:
This app doesn't work under Wayland. So I removed the Wayland permission. Now, this app only has `"--socket=x11"`.